### ISSUE TITLE
XMachine formatter does not indent the end keyword for events.

### ISSUE THREAD

**Issue by tshoang** [eventB-Soton/XEventB#30]
The formatter for XMachine does not indent the end keyword for events when there is a present of witness (``with`` clause) as showed in the following example.
```
	convergent event CloseCourse
	refines CloseCourses
	any
		c
	where
		@grd2_1: c ∈ dom(atnds)
	then
		@act1_2: atnds ≔ {c} ⩤ atnds
	with
		@cs: cs = {c} 
end
```

The formatter seems to work fine without the ``with`` clause as in the following example
```
	convergent event Register
	refines Register
	any
		p
		c
	where
		@grd2_1: p ∈ PRTCPT
		@grd2_2: p ≠ instrs(c)
		@grd2_3: c ∈ dom(atnds)
		@grd2_4: p ∉ atnds(c)
		theorem @thm2_3: atnds(c) = prtcpts[{c}]
	then
		@act2_1: atnds(c) ≔ atnds(c) ∪ {p}
	end
```

**Comment by dd4g12** [comment]
fixed

**Comment by tshoang** [comment]
For the record, this is closed by commit b3d0feba8e003eb829ac498101f6afbba0399c53.
